website/src/releases/2023-02/deployment-previews.mdx

```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-03-03
+title: Custom Applications deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [Deployment previews](../concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
```

Comment:
```suggestion
Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
```

Reply:
Nit: the sentence might be a bit confusing, as it seems that you can't use the production version without a preview.

```suggestion
Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features without affecting the production version of your application.
```